llvm/lib/DebugInfo/DWARF/DWARFVerifier.cpp

```diff
@@ -2026,12 +2028,35 @@ void OutputCategoryAggregator::EnumerateResults(
 }
 
 void DWARFVerifier::summarize() {
-  if (ErrorCategory.GetNumCategories() && DumpOpts.ShowAggregateErrors) {
+  if (!ErrorCategory.GetNumCategories())
+    return;
+  if (DumpOpts.ShowAggregateErrors) {
     error() << "Aggregated error counts:\n";
     ErrorCategory.EnumerateResults([&](StringRef s, unsigned count) {
       error() << s << " occurred " << count << " time(s).\n";
     });
   }
+  if (!DumpOpts.JsonSummaryFile.empty()) {
+    std::error_code EC;
+    raw_fd_ostream JsonStream(DumpOpts.JsonSummaryFile, EC, sys::fs::OF_Text);
+    if (EC) {
+      error() << "error opening aggregate error json file '"
```

Comment:
I'm guessing this might end up printing as `error: error opening ... ` which is a bit redundant - perhaps there's a better way to phrase it, maybe check how other file opening errors are reported in LLVM and by other command line tools (maybe check dwarfdump and llvm-dwarfdump on a non-existent file)

Reply:
'unable to open' definitely reads better.

Reply:
(That's how it's phrased elsewhere)